Components/cards/cards.css

```diff
@@ -0,0 +1,91 @@
+.card-vertical{
+    background-color: rgb(252, 252, 252);
+    border-radius: 2px;
+    width: 15rem;
+    height: 26.5rem;
+    box-shadow: 5px 5px 10px 0px rgb(151, 151, 151);
+    margin: 1rem;
+    position: relative;
+    cursor: pointer;
+}
+
+.card-img-vertical{
+    width: 15rem;
+    height: auto;
+}
+
+.card-heading{
+    font-weight: 500;
+    margin: 0 0rem 0 0.5rem;
+}
+
+.card-heading-2{
+    font-size: 0.65rem;
+    font-weight: 300;
+    margin: 0rem 0 0 0.5rem;
+    color: rgb(77, 77, 77);
+}
+
+.addtocart{
+    background-color: rgb(38, 40, 61);
+    text-align: center;
+    color: white;
+    width: 10rem;
+    padding: 0.5rem;
+    margin: auto;
+    border-radius: 3px;
+}
+
+.price{
+    margin: 0.2rem 0 0.75rem 0.5rem;
+}
+
+.card-badge{
+    position: absolute;
+    background-color: #ffa600;
+    color: rgb(38, 40, 61);
+    font-weight: 400;

```

Comment:
follow a single color code pattern dont mix rgb with hex

Reply:
Sure, will keep all in rgb